### ISSUE TITLE
left over bug(s) from #260

### ISSUE THREAD

**Issue by xtj-9182** [raulhaag/MiMangaNu#263]
So I accidentally closed #260 by using the fixed keyword even though I wanted to say it's only almost fixed. Anyway so while we don't crash anymore, there is still something weird going on with
Sexual Hunter Riot ch. 20 from KissManga. When I reset chapter ch. 20 and try to tap on the chapter I'm getting an Error toast and when I try to download it it won't start the download. However when I remove the chapter and update the list it will work somehow ... so something weird is going on with reset chapter on ch. 20.

**Comment by Johndeep** [comment]
This and other errors seems to happen, because some of the chapters does have weird name, mostly those with exotic UTF-8 Symbols, which always creates problems for the http request class.

**Comment by xtj-9182** [comment]
ok so after some investigation I found out that doing `setPages(0)` in the reset function is causing this but I have no idea why since I could only reproduce this with Sexual Hunter Riot (any chapter). I think it might have to do with the `GetPageTask` asynctask from `ActivityPagedReader` probably not related to the special chars but I'm not sure. For now I just commented out the `setPages(0)` call, we should probably improve this fix but for now it works and is better than showing a non descriptive error.
